perception/autoware_tensorrt_yolox/src/tensorrt_yolox.cpp

```diff
@@ -341,6 +348,17 @@ TrtYoloX::~TrtYoloX()
   }
 }
 
+bool TrtYoloX::setCudaDeviceId(const uint8_t gpu_id)
+{
+  cudaError_t cuda_err = cudaSetDevice(gpu_id);
+  if (cuda_err != cudaSuccess) {
+    std::cerr << "Failed to set CUDA device: " << cudaGetErrorString(cuda_err) << std::endl;
```

Comment:
@ismetatabay 
Thanks!
Can you replace this `std::cerr` to `RCLCPP_ERROR`, too?

Reply:
Thank you as well. Similarly, this function is included in TrtYoloX instead of TrtYoloXNode. Therefore, we cannot directly change it to use ROS logs. We can either remove it since we have already implemented the GPU initialization control and log mechanism in the ROS nodes, or we can keep it. I’m okay with both options. What do you think? @Shin-kyoto -san & @Owen-Liuyuxuan -san

Reply:
done :+1: